site/src/atomics/spacing.md

```diff
@@ -6,7 +6,70 @@ template: standard
 
 # Spacing Atomics
 
+Spacing atomics can be used to add margin or padding on elements. They are especially helpful when composing layouts.
+
+| cssproperty                                                                                                                                                                                                            | value                                                   | screensize                        |
+| ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------- | --------------------------------- |
+| `margin`, `padding`, `margin-block`, `padding-block`,`margin-inline`, `padding-inline`, `margin-top`, `padding-top`, `margin-right`, `padding-right`, `margin-bottom`, `padding-bottom`, `margin-left`, `padding-left` | `xxs`, `xs`, `s`, `m`, `l`, `xl`, `xxl`, `xxxl`, `none` | `tablet`, `desktop`, `widescreen` |
+
+## Usage
+
+_Adding uniform spacing_
+
+Use `margin` or `padding` with a size value to add spacing around an element.
+
+```html
+<div class="margin-xxs border">XXS</div>
+<div class="margin-xs border">XS</div>
+<div class="margin-s border">X</div>
+<div class="margin-m border">M</div>
+<div class="margin-l border">L</div>
+<div class="margin-xl border">XL</div>
+<div class="margin-xxl border">XXL</div>
+<div class="margin-xxxl border">XXXL</div>
+```
+
+_Adding horizontal and vertical spacing_
```

Comment:
Same as before, remove _Adding horizontal and vertical spacing_?

Reply:
The sentences you've got below to a good job of explaining how things work.

Reply:
Removed